timur/lib/client/jsx/components/query/query_results.tsx

```diff
@@ -81,6 +83,12 @@ const QueryResults = () => {
     setPage(newPage);
   }
 
+  const userColumnsStr = useMemo(() => {
+    return JSON.stringify(userColumns(columns));
+  }, [columns]);
+
+  const codeMirrorText = `query: ${queryString}\\nuser_columns: ${userColumnsStr}`;
```

Comment:
extra `\` here for the `\n` causes it all to render on one line for me.

Reply:
Oh phooey, thanks, I'll take another look. Without the extra `\` I was seeing funky test behavior. :stuck_out_tongue_closed_eyes: